Capstone_Project/BaseLine.ipynb

```diff
@@ -1,5783 +0,0 @@
-       "3  air_00a91d42b08b08d9          3         2  Tōkyō-to Chiyoda-ku Kudanminami   \n",
-       "4  air_00a91d42b08b08d9          6         5  Tōkyō-to Chiyoda-ku Kudanminami   \n",
-       "5  air_00a91d42b08b08d9         16         2  Tōkyō-to Chiyoda-ku Kudanminami   \n",
-       "6  air_00a91d42b08b08d9          2         3  Tōkyō-to Chiyoda-ku Kudanminami   \n",
-       "7  air_00a91d42b08b08d9          4         1  Tōkyō-to Chiyoda-ku Kudanminami   \n",
-       "8  air_00a91d42b08b08d9          1         2  Tōkyō-to Chiyoda-ku Kudanminami   \n",
-       "9  air_00a91d42b08b08d9         13         7  Tōkyō-to Chiyoda-ku Kudanminami   \n",
-       "\n",
-       "   air_genre_name   latitude   longitude  holiday_flg  Week_day  day  month  \\\n",
-       "0  Italian/French  35.694003  139.753595            0         3   14      1   \n",
-       "1  Italian/French  35.694003  139.753595            0         4   15      1   \n",
-       "2  Italian/French  35.694003  139.753595            0         5   16      1   \n",
-       "3  Italian/French  35.694003  139.753595            0         4   22      1   \n",
-       "4  Italian/French  35.694003  139.753595            0         4   29      1   \n",
-       "5  Italian/French  35.694003  139.753595            0         4    5      2   \n",
-       "6  Italian/French  35.694003  139.753595            0         1    8      3   \n",
-       "7  Italian/French  35.694003  139.753595            0         0    4      4   \n",
-       "8  Italian/French  35.694003  139.753595            0         3    7      4   \n",
-       "9  Italian/French  35.694003  139.753595            0         4    8      4   \n",
-       "\n",
-       "   year  fri  sat  sun  geo_location  \n",
-       "0  2016    0    0    0    175.447598  \n",
-       "1  2016    1    0    0    175.447598  \n",
-       "2  2016    0    1    0    175.447598  \n",
-       "3  2016    1    0    0    175.447598  \n",
-       "4  2016    1    0    0    175.447598  \n",
-       "5  2016    1    0    0    175.447598  \n",
-       "6  2016    0    0    0    175.447598  \n",
-       "7  2016    0    0    0    175.447598  \n",
-       "8  2016    0    0    0    175.447598  \n",
-       "9  2016    1    0    0    175.447598  "
-      ]
-     },
-     "execution_count": 232,
-     "metadata": {},
-     "output_type": "execute_result"
-    }
-   ],
-   "source": [
-    "#Add new features fri/Sat/Sun\n",
-    "#fri == 4; sat == 5; sun == 6\n",
-    "df_total['Week_day'] = df_total['visit_date'].dt.dayofweek\n",
-    "df_total['day'] = df_total['visit_date'].dt.day\n",
-    "df_total['month'] = df_total['visit_date'].dt.month\n",
-    "df_total['year'] = df_total['visit_date'].dt.year\n",
-    "df_total['fri'] = (df_total['Week_day'] == 4)*1\n",
-    "df_total['sat'] = (df_total['Week_day'] == 5)*1\n",
-    "df_total['sun'] = (df_total['Week_day'] == 6)*1\n",
-    "df_total['geo_location'] = (df_total['latitude'] + df_total['longitude'])\n",
```

Comment:
@nishalpattan : why adding latitude plus longitude? Why not considering the coordinates directly? that is (x,y)

Reply:
I am doubted about this feature, that both longitude and latitude contribute as a single feature as geo location.

Reply:
I might be wrong in this.